tests/gtest/test_place_holder2.cpp

```diff
@@ -0,0 +1,126 @@
+#include "gtest/gtest.h"
+#include "place_holder2.hpp"
+
+// # R code that generates true values for the test
```

Comment:
Can we remove the commented out code for now? If we want to include, should do so in a separate file.

Reply:
Those commented out code are for generating one-time true values for C++ unit tests. I agree that for complicated tests we need to put the R script somewhere else, do you have a suggestion on where to put the R script? For simple C++ unit tests, commented out R code is part of the testing documentation. The benefits of including the simple R code as comments in the same C++ testing file is that you can remove or move C++ tests without worrying about where the R code is. 